server/lib/gutenbach

```diff
@@ -283,10 +274,6 @@ sub resolve_external_reference {
     # Take the leading non-whitespace as a URL
     $uri=$1;
 
-    if (is_youtube_url($uri)) {
-      return $uri;
-    }
```

Comment:
That isn’t what I meant. We should unconditionally `return $uri` here and delete the remaining dead code.

Reply:
Is this equivalent to not calling https://github.com/sipb/gutenbach/blob/master/server/lib/gutenbach#L189 at all.

Reply:
No, `resolve_external_reference` isn’t a no-op—it takes the path to a file containing a URL, reads the file, and returns the URL.